docs/develop/add-in-manifests.md

```diff
@@ -2,23 +2,23 @@
 title: Office Add-ins manifest
 description: Get an overview of the Office Add-in manifest and its uses.
 ms.topic: overview
-ms.date: 07/27/2023
+ms.date: 03/15/2024
 ms.localizationpriority: high
 ---
 
 # Office Add-ins manifest
```

Comment:
There's a new Outlook on Windows, now in preview. Do we need to explicitly indicate support or lack thereof?

Reply:
> There's a new Outlook on Windows, now in preview. Do we need to explicitly indicate support or lack thereof?

Sam says "yes", so I'll add this.